tony-core/src/main/java/com/linkedin/tony/ApplicationMaster.java

```diff
@@ -718,6 +693,12 @@ private int doPreprocessingJob() throws Exception {
     extraEnv.put("HOME", System.getProperty("user.dir"));
     extraEnv.put(Constants.PY4JGATEWAY, String.valueOf(gatewayServerPort));
     String taskCommand = baseTaskCommand;
+    if (tonyConf.get(TonyConfigurationKeys.getExecuteCommandKey(Constants.AM_NAME),
+            tonyConf.get(TonyConfigurationKeys.getContainerExecuteCommandKey())) != null) {
+        taskCommand = tonyConf.get(TonyConfigurationKeys.getExecuteCommandKey(Constants.AM_NAME),
+                tonyConf.get(TonyConfigurationKeys.getContainerExecuteCommandKey()));
```

Comment:
I think this `if` block would be more readable if you used a local variable, e.g.:
```
String amCommand = tonyConf.get(..., tonyConf.get(...));
if (amCommand != null) {
  taskCommand = amCommand;
}
```

Reply:
removed this logic, taskCommand is always in conf after previous changes